### ISSUE TITLE
Discovery: visual focus indicator color contrast

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#5480]
## Description
This is a discovery ticket to identify how to solve [DST Staging Review: Visual focus indicator has insufficient color contrast against adjacent (background) color](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4683). 

## Tasks
- [ ] Research how to solve the visual focus indicator to meet color contrast requirements
- [ ] Share research with DST for feedback
- [ ] Write ticket with solution

## Acceptance Criteria
- [ ] Discovery completed
- [ ] Ticket for follow up work done
